### PR TITLE
v1.0.2-rc.1

### PR DESCRIPTION
**Pull request by ancientTexts-net**
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword/markdown-translatable/11)

### CODE REVIEW

**Comment by ancientTexts-net** [comment]
already published to NPM

**Comment by ancientTexts-net** [comment]
___
*[package.json, line 3 at r1](https://reviewable.io/reviews/unfoldingword/markdown-translatable/11#-MBEeSox0_Jtk28t0DRG:-MBEykGw5auISgme1ptf:b-bmox08) ([raw file](https://github.com/unfoldingword/markdown-translatable/blob/a8be815c0cabcdd3d2471287d97949d536262770/package.json#L3)):*
<details><summary><i>Previously, mandolyte wrote…</i></summary><blockquote>

Did you intend this to be a patch release? Or was this supposed to be rc.2? 

In other words, double check `1.0.2-rc.1` vs. `1.0.1-rc.2`
</blockquote></details>

I believe it is a patch because it fixes new bugs?

**Comment by ancientTexts-net** [comment]
1.0.1-rc.2 makes sense.  Thanks

**Comment by ancientTexts-net** [comment]
___
*[package.json, line 3 at r1](https://reviewable.io/reviews/unfoldingword/markdown-translatable/11#-MBEeSox0_Jtk28t0DRG:-MBF6Izr4Rcv5ATl4F1L:b-iafa63) ([raw file](https://github.com/unfoldingword/markdown-translatable/blob/a8be815c0cabcdd3d2471287d97949d536262770/package.json#L3)):*
<details><summary><i>Previously, ancientTexts-net (Zach Pennington) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

1.0.1-rc.2 makes sense thanks
